static/app/stores/groupStore.tsx

```diff
@@ -315,9 +316,13 @@ const storeConfig: GroupStoreDefinition = {
   },
 
   // TODO(dcramer): This is not really the best place for this
-  onAssignToError(_changeId, itemId, _error) {
+  onAssignToError(_changeId, itemId, error) {
     this.clearStatus(itemId, 'assignTo');
-    showAlert(t('Unable to change assignee. Please try again.'), 'error');
+    if (error.responseJSON?.detail === 'Cannot assign to non-team member') {
+      showAlert(t('Cannot assign to non-team member'), 'error');
```

Comment:
This is where the frontend message is changed to match the backend message `Cannot assign to non-team member`, if this is the error that occurred